gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -145,6 +141,9 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         .getFeatureSelections()
         .as(autoDisposable(this))
         .subscribe(this::onFeatureSelection);
+    mapContainerViewModel
+        .getAddFeatureButtonClicks()
```

Comment:
Could you also replace the click stream with an Observable<> instead of a LiveData<Action>? `ifUnhandled` relies on side effect to achieve the same result and will eventually be removed.